src/client/web/src/components/BIOMD/BIOMD/BIOMD-UI/UI-Landing.vue

```diff
@@ -40,8 +40,6 @@
 <script setup>
 import Card from "../BIOMD-UI/UI-Card.vue";
 
-import { ref } from "vue";
```

Comment:
Why was `ref` removed here?

Reply:
ref was unused in that component @irushavm